src/device-registry/utils/create-event.js

```diff
@@ -710,6 +710,10 @@ const createEvent = {
             let update = event.update;
             dot.delete(["filter", "update", "options"], value);
             update["$push"] = { values: value };
+            /**
```

Comment:
just as reminder for some other modifications we could make while storing measurements